### PR TITLE
Completed Design-2 Please review

### PR DESCRIPTION
**Pull request by arzoouzma01**

### CODE REVIEW

File: Problem_2.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/*
+   
+Design Hash-set
+*/
+
+//Time comlpexity - O(log n/k)
```
Comment (super30admin) [inline Problem_2.java:6]: Which part is making it O(log n/k). As discussed in class time complexity is O(1). Please watch video again

**Comment by super30admin** [comment]
Kindly review inline comment. Make amendments and resubmit

**Comment by super30admin** [comment]
Kindly review inline comment. Make amendments and resubmit

**Comment by arzoouzma01** [comment]
I will make the amendments and resubmit. I didn't use the approach used in the video but I will check the video again and then implement.

**Comment by arzoouzma01** [comment]
new edit

**Pull request closed without merging**